packaging/conf/configuration.yml

```diff
@@ -35,9 +35,6 @@
 #################################################################
 
 default:
-  rails_cache_store: <%= ENV.fetch('RAILS_CACHE_STORE') { :memcache }.to_sym %>
-  session_store: <%= ENV.fetch('SESSION_STORE') { :active_record_store }.to_sym %>
```

Comment:
SESSION_STORE is never set by packager. So if it is used by someone without prefix, they can just replace it themselves